### PR TITLE
[ci] Add jupyter notebook to pytest

### PR DESCRIPTION
**Pull request by neozhaoliang**
This PR will add a test case for running Taichi in IPython notebooks.

### CODE REVIEW

**Review by ailzhang — CHANGES_REQUESTED** [review]
Could you also provide a link or screenshot demonstrating this test runs in CI? I couldn't find it in https://pipelines.actions.githubusercontent.com/serviceHosts/ebf0abad-1193-4ff9-9c60-da990d1eacd9/_apis/pipelines/1/runs/55924/signedlogcontent/17?urlExpires=2022-11-17T02%3A53%3A55.4197393Z&urlSigningMethod=HMACV1&urlSignature=KZPg8LIrMsavleViQkzN6mISvhT0UzpWu26wZ6i4TEc%3D

File: tests/python/test_ipython.ipynb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "83062572",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import taichi as ti\n",
+    "\n",
+    "def test_kernel_print():\n",
```
Comment (ailzhang) [inline tests/python/test_ipython.ipynb:20]: IIUC you only defined a kernel but didn't actually run it right?

Comment (neozhaoliang) [reply tests/python/test_ipython.ipynb:20]: Doesn't pytest automatically collect all functions begin with `test_xxx` and run it? :thinking: :

Comment (neozhaoliang) [reply tests/python/test_ipython.ipynb:20]: Yes, it didn't actually run the function. Should be fixed now.

**Pull request closed without merging**